Makefile

```diff
@@ -317,6 +317,12 @@ mdlint-ctr:
 
 .PHONY: clean
 clean:
+	-sudo pkill '^socket_vmnet'
```

Comment:
This might abruptly stop finch installed via installer and create a bad state for the version installed by brew/installer. It might be safer to kill the `qemu `and `limactl` pinned in this repo. Something like ` pkill <Install directory>/finch/_output/lima/bin/qemu-system-x86_64`

Reply:
It will be anyway impacted since the shared folder is removed i.e., /opt/finch/.
The user anyway need to remove the vm and re-init it in this case.